components/rdv/DeleteRdvModal.tsx

```diff
@@ -37,7 +46,10 @@ export default function DeleteRdvModal({
 
       {aDesJeunesDUnAutrePortefeuille && (
         <div className='text-base-regular text-content_color text-center mx-28 mt-12'>
-          <p>Le créateur recevra un email de suppression de l’événement.</p>
+          <p>
+            Le créateur recevra un email de suppression{' '}
```

Comment:
```suggestion
            Le créateur recevra un e-mail de suppression{' '}
```
cf lama system et règle glossaire, en français